### DISCUSSION TITLE
Figma

### DISCUSSION THREAD

**Discussion by iparthenios** [sdy60/syllabus#103] [Q&A] [answered]
Μήπως έχει δει κάποιος το Figma για σχεδιασμό οθονών και prototyping?

**Comment by std133524** [comment, marked as answer]
Προσωπικά όχι, το έχω ακούσει αλλά δεν το έχω χρησιμοποιήσει. Υπάρχει και το  https://www.protopie.io/. 
Εδώ μια σύγκριση μεταξύ των δυο: https://www.theturnsignalblog.com/blog/guide-to-prototyping-automotive-interfaces/.
" it has all the prototyping features of Figma but with much more advanced options for variables, animations, voice interaction, and dynamic data."
